stripe/stripe_object.py

```diff
@@ -152,13 +187,13 @@ def __delitem__(self, k):
     # Custom unpickling method that uses `update` to update the dictionary
     # without calling __setitem__, which would fail if any value is an empty
     # string
-    def __setstate__(self, state):
+    def __setstate__(self, state: Dict[str, Any]) -> None:
         self.update(state)
 
     # Custom pickling method to ensure the instance is pickled as a custom
     # class and not as a dict, otherwise __setstate__ would not be called when
     # unpickling.
-    def __reduce__(self):
+    def __reduce__(self) -> Any:
```

Comment:
```suggestion
    def __reduce__(self) -> Tuple[type, Tuple, dict]:
```

Reply:
`Tuple` needs type variables, so that won't quite work.

We can do 
```
    def __reduce__(self) -> Tuple[Any, ...]:
``` though, that's close to what they have in typeshed: https://github.com/python/typeshed/issues/6661

Reply:
Fixed in [Address code review](https://github.com/stripe/stripe-python/pull/1088/commits/067ab1bfd6a74acd7cb4bb8c4f7e878a2d0e4661)